src/goto-programs/goto_convert.cpp

```diff
@@ -2133,6 +2230,24 @@ void goto_convertt::collect_operands(
 
 /*******************************************************************\
 
+Function: is_size_one
+
+  Inputs: Goto program 'g'
+
+ Outputs: True if 'g' has one instruction
+
+ Purpose: This is (believed to be) faster than using std::list.size
+
+\*******************************************************************/
+
+static inline bool is_size_one(const goto_programt &g)
```

Comment:
No idea, inherited it from the original code -- suggest asking @kroening about this as he was the original author